CHANGELOG

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [2.0.25] - 2014-10-03
+- Workaround for StoRM in the Domain Foreign Keys Test (GRIDINFO-58) (Maria Alandes)
```

Comment:
```suggestion
### Fixed
- Workaround for StoRM in the Domain Foreign Keys Test (GRIDINFO-58) (Maria Alandes)
```

Just trying to meet the Keep A Changelog format

Reply:
Great, thanks! Yes there is room for improvement on that area :)